ddl/job_table.go

```diff
@@ -220,6 +223,108 @@ func (d *ddl) delivery2worker(wk *worker, pool *workerPool, job *model.Job) {
 	})
 }
 
+// getDDLReorgHandle get ddl reorg handle.
+func getDDLReorgHandle(sess *session, job *model.Job) (element *meta.Element, startKey, endKey kv.Key, physicalTableID int64, err error) {
+	sql := fmt.Sprintf("select ele_id, ele_type, start_key, end_key, physical_id from mysql.tidb_ddl_reorg where job_id = %d", job.ID)
+	rows, err := sess.execute(context.Background(), sql, "get_handle")
+	if err != nil {
+		return nil, nil, nil, 0, err
+	}
+	if len(rows) == 0 {
+		return nil, nil, nil, 0, meta.ErrDDLReorgElementNotExist
+	}
+	id := rows[0].GetInt64(0)
```

Comment:
It is better to check the length of the row first.

Reply:
the `row` is the recordset of sql `select ele_id, ele_type, start_key....`, it must have enough length